setup.cfg

```diff
@@ -53,3 +53,13 @@ exclude = (?x)(
   | dask_cudf/dask_cudf/tests/
   # This close paren cannot be in column zero otherwise the config parser barfs
  )
+
+[codespell]
+# note: pre-commit passes explicit lists of files here, which this skip file list doesn't override -
+# this is only to allow you to run codespell interactively
+skip = ./.git,./.github,./cpp/build,.*egg-info.*,versioneer.py,./.mypy_cache,./cpp/tests,./python/cudf/cudf/tests,./java/src/test,./cpp/include/cudf_test/cxxopts.hpp
+# ignore short words, and typename parameters like OffsetT
+ignore-regex = \b(.{1,4}|[A-Z]\w*T)\b
+ignore-words-list = inout,unparseable
```

Comment:
ignore list is so small.
In cspell, I had 74 words for cudf, 117 words for C++14 keywords.
code-spell is non-dictionary based. misspellings checker!

Reply:
thanks @karthikeyann !   I've applied your suggestions in the last commit.

One reason I like codespell is because of the relatively small number of false positives, meaning the exclusions is relatively small compared to other tools -